### PR TITLE
Make all DAO screen tables resize to max. screen height

### PR DESCRIPTION
**Pull request by chimp1984**

### CODE REVIEW

**Comment by ripcurlx** [comment]
Just a heads up that I haven't planed to be available until 2nd of Jan, but I'll still try to do some reviews tomorrow.

**Comment by ripcurlx** [comment]
Just a heads up that I haven't planed to be available until 2nd of Jan, but I'll still try to do some reviews tomorrow.

File: desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java [hunk 1]
```diff
@@ -460,10 +461,13 @@ private void createCyclesTable() {
         GridPane.setRowIndex(cyclesTableView, gridRow);
         GridPane.setMargin(cyclesTableView, new Insets(Layout.FIRST_ROW_AND_GROUP_DISTANCE, -10, -15, -10));
         GridPane.setColumnSpan(cyclesTableView, 2);
+        GridPane.setVgrow(cyclesTableView, Priority.SOMETIMES);
         root.getChildren().add(cyclesTableView);
 
         cyclesTableView.setItems(sortedCycleListItemList);
         sortedCycleListItemList.comparatorProperty().bind(cyclesTableView.comparatorProperty());
+
+
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java:470]: Nit: Unnecessary new lines

**Review by sqrrm — APPROVED** [review]
utACK

**Pull request merged**